PlayCover/Views/MainView.swift

```diff
@@ -226,3 +302,98 @@ struct MainView_Previews: PreviewProvider {
             .environmentObject(AppIntegrity())
     }
 }
+
+struct AddFolderView: View {
+    @State var newFolder = ""
+    @Binding var addFolderSheet: Bool
+    var body: some View {
+        VStack {
+            TextField(text: $newFolder, label: {Text("preferences.textfield.url")})
+            Spacer()
+                .frame(height: 20)
+            HStack {
+                Spacer()
+                Button {
+                    addFolderSheet.toggle()
+                } label: {
+                    Text("button.Cancel")
+                }
+                Button {
+                    addFolderSheet.toggle()
+                } label: {
+                    Text("button.OK")
+                }
+                .tint(.accentColor)
+                .keyboardShortcut(.defaultAction)
+            }
+        }
+        .padding()
+        .frame(width: 600, height: 100)
+    }
+}
+
+struct Folder: Identifiable, Codable {
+    let id: UUID
+    var name: String
+    var apps: [String]
+    var icon: String = "folder"
+    init(name: String, icon: String) {
+        self.id = UUID()
```

Comment:
`id` and `apps` should be in-line initialization since they won't change between different initializations.